### PR TITLE
Frontend-tiles bug fix and argument clean-up

### PR DESCRIPTION
**Pull request by eringram**
- Fixed an incorrect property name in `queryGraphicRepresentations`
- Changed `obtainMeshExportTilesetUrl` function arguments to directly take the  iTwin, iModel, and changeset IDs instead of an `IModelConnection` object
    - The reason for this is that sometimes it's desirable to call this function before an `IModelConnection` is created, and the function just uses the connection to get the IDs anyway
- Moved logger initialization earlier during the Display Test App startup. I noticed I was missing useful info logged before `Logger.initializeToConsole` was called.

### CODE REVIEW

File: extensions/frontend-tiles/src/GraphicsProvider/GraphicsProvider.ts [hunk 1]
```diff
@@ -8,12 +8,12 @@ import { IModelApp, IModelConnection} from "@itwin/core-frontend";
...
 import { loggerCategory} from "../LoggerCategory";
 
-/** Arguments supplied  to [[obtainMeshExportTilesetUrl]].
+/** Arguments supplied  to [[obtainIModelTilesetUrl]].
  * @beta
  */
 export interface ObtainIModelTilesetUrlArgs {
   /** The iModel for which to obtain a tileset URl. */
-  iModel: IModelConnection;
+  iModel: IModelConnection | { iTwinId: string, iModelId: string, changesetId: string };
```
Comment (jason-crow) [inline extensions/frontend-tiles/src/GraphicsProvider/GraphicsProvider.ts:16]: `changesetId?: string` 
^^ optional right?

Comment (eringram) [reply extensions/frontend-tiles/src/GraphicsProvider/GraphicsProvider.ts]: Yeah it should be, fixed

**Comment by jason-crow** [comment]
https://github.com/Mergifyio backport release/4.8.x

**Review by markschlosseratbentley — APPROVED** [review]

**Review by jason-crow — APPROVED** [review]

**Pull request merged**